www/js/app.js

```diff
@@ -1456,6 +1456,27 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
                     // Configure home key press to focus #prefix only if the feature is in active state
                     if (params.useHomeKeyToFocusSearchBar)
                         iframeArticleContent.contentWindow.addEventListener('keydown', focusPrefixOnHomeKey);
+                    // Add event listener to iframe window to check for links to external resources
+                    iframeArticleContent.contentWindow.addEventListener('click', function (e) {
+                        if (e.target.tagName === 'A') {
+                            var href = e.target.getAttribute('href');
+                            // We assume that, if an absolute http(s) link is hardcoded inside an HTML string,
+                            // it means it's a link to an external website.
+                            // We also do it for ftp even if it's not supported any more by recent browsers...
+                            if (/^(?:http|ftp)/i.test(href)) {
+                                var target = e.target.getAttribute('target');
+                                // We don't override the default browser behavior if there's already a target attribute
```

Comment:
Actually, I'm not sure about this. If I ask to be warned before the browser opens a link on the Internet (albeit in a new tab), I would expect to be warned uniformly in all cases, not only if the link happens not to have a target. So I think the warning should happen anyway, just don't set the target if it is already set. What do you think @mossroy ?

I have tested the functionality, and it works well.

Reply:
Based on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target , the target can be used to communicate between tabs/windows/frames/iframes etc
I thought it might be safer to not do anything in such cases.
But you're right: we're talking about the case of an external target URL, for which it should be better to warn the user anyway.
I'll change accordingly